packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -3674,6 +3931,9 @@ function pushStartCustomElement(
     }
   }
 
+  // TODO: ViewTransition attributes gets observed by the Custom Element which is a bit sketchy.
+  pushViewTransitionAttributes(target, formatContext);
```

Comment:
yeah but isn't this still a plus for styling?

Reply:
What do you mean? The issue is that custom elements are likely to observe any attributes you set on them as their private name space so the extension is more problematic on those.